src/js/components/Map/map.js

```diff
@@ -0,0 +1,46 @@
+export default class Map {
+    create() {
+        const mapOptions = {
+            center: [17.385044, 78.486671],
+            zoom: 4
+        };
+
+        const map = new L.map('map', mapOptions);
+        const layer = new L.TileLayer('https://{s}.tile.thunderforest.com/spinal-map/{z}/{x}/{y}.png');
+
+        async function mapEffect({ leafletElement: map } = {}) {
+            let response;
+
+            try {
+                response = await axios.get('https://corona.lmao.ninja/v2/countries');

```

Comment:
Also, why do you need the axios library? You used simple fetch in some files but now there is axios. If you use axios for async requests you should use it everywhere